dev/com.ibm.ws.concurrent.mp.1.0/src/com/ibm/ws/concurrent/mp/ConcurrencyManagerImpl.java

```diff
@@ -79,6 +79,12 @@
         contextProviders.add(concurrencyProvider.wlmContextProvider);
         available.add(WLMContextProvider.WORKLOAD);
 
+        ThreadContextProvider cdiCtx = concurrencyProvider.cdiContextProvider;
+        if (cdiCtx != null) {
```

Comment:
Should treat CDI context similar to the other context types, per offline discussion

Reply:
will fix this in a follow-on PR